corehq/apps/app_manager/views/download.py

```diff
@@ -54,13 +56,15 @@ def download_odk_media_profile(request, domain, app_id):
     if not request.app.copy_of:
         username = request.GET.get('username', 'unknown user')
         make_async_build.delay(request.app, username)
+    else:
+        request._always_allow_browser_caching = True
     return HttpResponse(
         request.app.create_profile(is_odk=True, with_media=True),
         content_type="commcare/profile"
     )
 
 
-@safe_download
+@safe_cached_download
```

Comment:
@calellowitz Example of one of the make new build from mobile feature. This checks if the app is a copy of another and then tries to get the latest app. I think this would only be hit if there was not an app previously though right? in that case caching should be OK I think